ntirpc/rpc/work_pool.h

```diff
@@ -72,6 +72,7 @@ struct work_pool_thread {
 	char worker_name[16];
 	pthread_t pt;
 	uint32_t worker_index;
+	bool wakeup;
```

Comment:
A comment on why and how it is used should be good.

Reply:
I added the comments where I am using it, let me know if thats useful. I am not sure adding comment to the structure will help.